services/enrichment/manager/src/events/onFileIngested.js

```diff
@@ -26,6 +26,9 @@ export const handler = async (event, context, callback) => {
   // Get Account ID from lambda function arn in the context
   const accountId = context.invokedFunctionArn.split(':')[4];
 
+  const sns = new AWS.SNS();
```

Comment:
Again a very small thing - clients instantiations (s3, sns, etc.) could be placed next to each other, either in the beginning of the function or in the `try`. Easier to follow later, at least for me.